client/state/themes/themes/test/reducer.js

```diff
@@ -158,4 +159,38 @@ describe( 'themes reducer', () => {
 			expect( state ).to.eql( fromJS( jsObject ) );
 		} );
 	} );
+
+	describe( 'themes received', () => {
+		const siteId = 12345678;
+		const twentyfifteen = Map( {
+			name: 'Twenty Fifteen',
+			id: 'twentyfifteen'
+		} );
+		const twentysixteen = {
+			name: 'Twenty Sixteen',
+			id: 'twentysixteen'
+		};
+		const state = fromJS( {
+			themes: {
+				twentyfifteen
+			},
+			siteId: 87654321
+		} );
+		const newState = reducer( state, {
+			type: THEMES_RECEIVE,
+			themes: [
+				twentysixteen
+			],
+			siteId: siteId
```

Comment:
An array key/value pair like this could be shortened to just

`siteId`

which thanks to ES6 does the right thing :-)

(Since this is a test and that const is only used twice, I'd actually opt for inlining its value, but that's a minor style nitpick.)